bower.json.erb

```diff
@@ -2,7 +2,7 @@
   "name": "rad-consumer",
   "dependencies": {
     "dough": "<%= gem_path('dough-ruby') %>",
-    "yeast": "git://github.com/moneyadviceservice/yeast.git#master",
+    "yeast": "git://github.com/moneyadviceservice/yeast.git#a1dad29959e057aae926f92d61b55dd519bed812",
```

Comment:
We're locked to a certain commit of dough for some reason (there's even an old issue about it: https://github.com/moneyadviceservice/rad_consumer/issues/179). Something was moved from yeast to dough but since we don't have the latest dough we needed this in order not to break